fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -671,6 +674,10 @@ public static AuditEventProcessor getCurrentAuditEventProcessor() {
         return getCurrentCatalog().getAuditEventProcessor();
     }
 
+    public EncryptKeyManager getEncryptKeyManager() {
+        return encryptKeyManager;
```

Comment:
This also need to be persisted in `saveImage()` of Catalog, and read it in `loadImage()`